sdk/storage/Azure.Storage.Files/tests/FileClientTests.cs

```diff
@@ -690,12 +690,7 @@ public async Task ListHandles()
             using (this.GetNewFile(out var file))
             {
                 // Act
-                var handles =
-                    (await file.GetHandlesAsync()
-                    .ByPage(pageSizeHint: 5)
-                    .ToListAsync())
-                    .SelectMany(p => p.Values)
-                    .ToList();
+                var handles = await file.GetHandlesAsync().ToListAsync();
```

Comment:
Yeah...  I left some of the `ByPage` calls to keep the tests as close as possible, but this one was a bit much.